front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm.tsx

```diff
@@ -261,9 +270,14 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
               },
             });
             if (!isNilOrError(projectFolder)) {
-              const imagesToAddPromises = projectFolderImages.map((file) =>
-                addProjectFolderImage(projectFolder.id, file.base64)
-              );
+              const imagesToAddPromises = projectFolderImagesBase64
+                ? projectFolderImages.map((file) =>
+                    addProjectFolderImage(
+                      projectFolder.id,
+                      projectFolderImagesBase64
+                    )
+                  )
+                : [];
```

Comment:
This is a bit odd, you're not using file anymore, so why would you map over projectFolderImages in the first place?